cmd/thor/node/packer_loop.go

```diff
@@ -204,6 +207,10 @@ func (n *Node) pack(flow *packer.Flow) error {
 		if v, updated := n.bandwidth.Update(newBlock.Header(), time.Duration(realElapsed)); updated {
 			log.Debug("bandwidth updated", "gps", v)
 		}
+
+		metricBlockProcessedTxs().Add(int64(len(receipts)))
+		metricBlockProcessedGas().Add(int64(newBlock.Header().GasUsed()))
+		metricBlockProcessedDuration().Observe(time.Duration(realElapsed).Milliseconds())
```

Comment:
Does this distinguish between tx metrics from packed blocks and tx metrics from received blocks ?

Reply:
No, I would consider the proposed block count as necessary but tx and gas are not.

Reply:
Why not? Would it not be interesting to see the tx count and gas sum of proposed blocks ?